### ISSUE TITLE
Skipping active spawn points

### ISSUE THREAD

**Issue by MemoriesofNemo** [mchristopher/PokemonGo-DesktopMap#1161]
I have 22 accounts running a 13 radius scan, and it's very fast. THe problem is though, it will often skip over a green spawnpoint without showing the pokemon there. I have not excluded any pokemon, so i worry that i will miss something amazing because of this. Has anyone else experienced this, or does someone have any advice?

**Comment by MemoriesofNemo** [comment]
To add on to this, the first rotation of scans always works perfectly. But subsequent loops will miss more and more spawns as time goes on, until a reboot of the program is needed, which brings it back up to a 100%  :/

**Comment by boggieboy** [comment]
I also seem to be having this issue, although I'm currently only running two accounts on a 5 radius scan. Everything scans fine at first with all the scanned locations consistently green, then a few spots will come up red after an hour or two before some spots start getting seemingly dropped altogether.

**Comment by boggieboy** [comment]
![untitled](https://cloud.githubusercontent.com/assets/21984082/18231017/a19cbc0a-72eb-11e6-8f34-f3ee75aa8e9b.png)

My map after scanning for a couple hours.

**Comment by boggieboy** [comment]
![untitled1](https://cloud.githubusercontent.com/assets/21984082/18231120/b3c52982-72ee-11e6-8a53-114ec988ec27.png)

Half hour later. As you can see, even more points have been dropped.

**Comment by MemoriesofNemo** [comment]
Well, my problem is slightly different, what i meant was the spawn points themselves are being skipped, not the scanned areas. So if theres a green spawnpoint, and the scanner is about to scan that area, it will scan the area but not reveal the pokemon. And the longer it has been scanning for, the more frequent it will do this

**Comment by boggieboy** [comment]
Sorry, my bad. I misunderstood. As far as that goes though, I have spawn points that are green or red but not showing up pokemon too. You can see a few in my above screenshots. I assumed a spawn point was not guaranteed to get a spawn every hour, but I could be wrong. Has this been tested?

**Comment by MemoriesofNemo** [comment]
No problem, but yes in my experience, a pokemon is guaranteed to spawn when the spawnpoint is active. Like i said, restarting the program will make it reveal the pokemon at the spawnpoints that it skipped over before restarting. Seems to me there are a couple bugs to iron out in this version hehe

**Comment by boggieboy** [comment]
Sorry, just checked and my spawn markers are all 30mins off from the actual spawn times for some reason -probably something to do with my timezone- so disregard my screenshots for that issue as they don't show the true spawn times.

**Comment by romanticheart** [comment]
Do you have the more common Pokemon hidden from the maps? Pidgey, Weedle, Rattata, etc?

**Comment by MemoriesofNemo** [comment]
No pokemon were excluded. I made sure, as this was my first suspicion aswell. 
The program simply dies out slowly over the course of 1.5-2 hours it seems, if i don't restart it. 

I figured out however, that if i run the program with the same accounts on a different pc with a different API (not sure if this makes a difference), it works perfectly fine, which leads me to believe it might be system related. 

The specs for the pc, where the bug occurs, are:
AMD Athlon(tm) II X4 640 Processer 3GHz
4 GB ram
32-bit

Specs for the working system are:
Intel(R) core(TM) i5-4670 CPU @ 3.40GHz
8,00 GB ram
64-bit
